README.md

```diff
@@ -1,6 +1,7 @@
 
 # no-OS
 
+[![Build Status](https://travis-ci.org/analogdevicesinc/no-OS.svg?branch=master)](https://travis-ci.org/analogdevicesinc/no-OS) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/8b54ad19266248f786280925a5967965)](https://www.codacy.com/manual/antoniumiclaus/no-OS?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=analogdevicesinc/no-OS&amp;utm_campaign=Badge_Grade)
```

Comment:
antoniumiclaus/no-OS ?

Reply:
In order to have access to configurations inside Codacy/ access badges, the repo needs to be linked to an account (the @analogdevicesinc organization is not added, someone with the proper rights has to do it). Before it was linked to @commodo 's account. (all Codacy checks for no-os repository were redirecting to something like https://app.codacy.com/manual/commodo/no-OS/pullRequest?prid=4584820)

This badge approach is used for other ADI repositories, please see: https://github.com/analogdevicesinc/pyadi-iio/commit/5d3bed43f0fd025c37a7fa527069edca85623cac#diff-04c6e90faac2675aa89e2176d2eec7d8R4